src/main/java/dev/socketmods/socketnukes/client/screen/ExploderConfigScreen.java

```diff
@@ -42,19 +44,22 @@ protected void init() {
 
         for(RegistryObject<ExtendedExplosionType> explosion : SNRegistry.EXPLOSIONS.getEntries()) {
             addButton(new Button(middleX + 10, topY - rollingOffset, 160, 20,
-                    new TranslationTextComponent(explosion.get().getRegistryName().getNamespace() + ".explosions." + explosion.get().getRegistryName().getPath()),
-                    button -> config(explosion.get().getRegistryName())));
+                    new TranslationTextComponent(Objects.requireNonNull(explosion.get().getTranslationText().getKey())),
+                    button -> config(Objects.requireNonNull(explosion.get().getRegistryName()))));
             rollingOffset += 30;
         }
     }
 
     @Override
     public void render(MatrixStack matrixStack, int mouseX, int mouseY, float partialTicks) {
+        assert this.minecraft != null;
         this.minecraft.getTextureManager().bindTexture(bg_texture);
         super.render(matrixStack, mouseX, mouseY, partialTicks);
     }
 
     private void config(ResourceLocation registryName) {
+        assert minecraft != null;
+        assert minecraft.player != null;
```

Comment:
This is called after init, the assertions are in place for Intellij, I'm leaving these as such for the moment. This is really a style issue which will need further discussion.

Reply:
From what I recall, assertions are disabled by default, which means these assertions will not work until the appropriate command line options are passed in. 

So, we either have to rely on these assertions as guides instead of checks (to _guide_ IntelliJ that these variables, while nullable, is impossible to be `null` in these situations), or to move them to `requireNonNull` checks instead.

In my opinion, unless it can be proven that it is impossible for `minecraft` and `minecraft.player` to be `null` in these circumstances, that these should be moved into `requireNonNull` checks instead.

Reply:
I agree normally `requireNonNull` is the way to go, however this is for GUI/Screen which I tend to use `assert`'s only to guide IntelliJ due to the design of the Screen system.

During object creation / constructor, `minecraft` is null.
Then the `init(Minecraft minecraft, int width, int height)` is called that sets it to a nonnull value
This then calls `init()` for you to setup your `Widget`s, `minecraft` is guaranteed to be nonnull.
`render` also should only ever be called after `init(3)` (making `minecraft` nonnull) is called as that sets the screen sizes for layout

Mojang tend to assume `minecraft` is nonnull with no nullability checks in these functions.

Reply:
In that case, I'd recommend leaving these as assertions and leaving a comment there to briefy explain (`// Assume that these are always non-null`).